### DISCUSSION TITLE
[Q&A] No Collect job enabled or Collect support disabled server side.

### DISCUSSION THREAD

**Discussion by D-Hawk** [glpi-project/glpi-agent#722] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Bonjour,

Dans un lab pour un projet futur à mon travail j'ai installé Glpi + l'agent sur un pc client, et je souhaite déployer un paquet (7ZIP pour le test)
J'ai ces messages qui reviennent à chaque fois : No Collect job enabled or Collect support disabled server side + No Deploy job enabled or Deploy support disabled server side.

Je vous mets le fichier de log ci dessous :

```
[Wed Jul 24 09:42:40 2024][info] target server0: server http://192.168.1.10/plugins/glpiinventory/
[Wed Jul 24 09:42:40 2024][info] sending prolog request to server0
[Wed Jul 24 09:42:41 2024][info] server0 answer shows it supports GLPI Agent protocol
[Wed Jul 24 09:42:42 2024][info] running task Collect
[Wed Jul 24 09:42:42 2024][info] No Collect job enabled or Collect support disabled server side.
[Wed Jul 24 09:42:43 2024][info] running task Inventory
[Wed Jul 24 09:42:43 2024][info] New inventory from PC.LAB.LOCAL-2024-07-14-13-39-19 for server0
[Wed Jul 24 09:43:06 2024][info] running task Deploy
[Wed Jul 24 09:43:06 2024][info] No Deploy job enabled or Deploy support disabled server side.
[Wed Jul 24 09:43:06 2024][info] target server0: next run: Wed Jul 24 21:10:26 2024 - http://192.168.1.10/plugins/glpiinventory/
[Wed Jul 24 09:43:06 2024][info] GLPI Agent memory usage: WSS=3469312 PFU=132993024

```

Le firewall est désactivé sur la VM du serveur GLPI, de même sur le PC client , côté serveur c'est Win SRV 2022 avec IIS, côté client Windows 10.
GLPI en 10.0.16, agent en 1.7.3

Dans la conf de l'agent, voici la clé registre avec l'url du serveur à joindre : `http://192.168.1.10/plugins/glpiinventory/`

J'ai regardé plusieurs posts sur internet et rien n'a résolu mon pb, si quelqu'un a une idée ?

Je précise que mon site est directement joignable via l'IP, je n'ai pas besoin de rajouter un /glpi dans mon url.

Merci d'avance pour vos réponses

**Comment by g-bougard** [comment]
Hello @D-Hawk 

firstly one comment, that's not a good way of doing to mix an english title with a french description. Also just use english as most people should understand this language in the community.

About Collect task, the message just tells you there's no task with the prepared status in GLPI for your agent. This can occur for different reason: no task is defined, the task is just not prepared in GLPI, your agent is not targeted by your task, the agent in GLPI has not collect task enabled.
Anyway don't forget to check this official faq article as this can help you (a french version exists) to find what you may have missed: https://faq.teclib.com/03_knowledgebase/inventory/Collect_data/
